lib/routers/task/index.js

```diff
@@ -30,6 +30,15 @@ router.put('/:taskId/status', (req, res, next) => {
     .catch(next);
 });
 
+router.post('/:taskId/comment', (req, res, next) => {
+  return req.workflow.task(req.taskId).comment({ comment: req.body.data, meta: pick(req.body.meta, 'comment') })
```

Comment:
I don't think the parameters you're sending here are correct. The comment should be the `comment` parameter, not part of the meta payload.

I'd envisage a client sending a request body of the form:

```
{
  comment: 'The comment content',
  meta: {
    optional: 'metadata'
  }
}
```

which would make this line:

```
return req.workflow.task(req.taskId).comment({ comment: req.body.comment, meta: req.body.meta })
```

Reply:
done